LibreTransmitterUI/LibreTransmitterManager+UI.swift

```diff
@@ -57,7 +57,8 @@ extension LibreTransmitterManagerV3: CGMManagerUI {
         let hostedView = DismissibleHostingController(
             content: settingsView
                 .navigationTitle(self.localizedTitle)
-                .environmentObject(displayGlucosePreference)
+                .environmentObject(displayGlucosePreference),
+            guidanceColors: GuidanceColors()
```

Comment:
not sure if `GuidanceColors()` should be used here instead of `.default`. I see the base assets do define the `warning` color, which may not match `.yellow`.

Reply:
ah... just noticed the `colorPalette` is passed in. This should use 

```
init (
        content: Content,
        dismissalMode: DismissalMode = .modalDismiss,
        isModalInPresentation: Bool = true,
        onDisappear: @escaping () -> Void = {},
        colorPalette: LoopUIColorPalette
    )
```

and just pass the colorPalette along.

Reply:
It is confusing that there is a convenience initializer for `DismissibleHostingController` that provides granular configuration of the colors, but it seems like we should never use it in practice. It seems like it exists for mock view controllers.